src/components/Dashboard/styles/PrintDashboard.module.css

```diff
@@ -14,12 +13,17 @@
     width: 1102px;
 }
 
+@media only screen and (max-width: 480px) {
+    .container {
+        padding-left: var(--spacers-dp12);
+    }
+}
+
 @media print {
-    .wrapper {
+    .container {
         background-color: white;
         height: auto !important;
         overflow: visible !important;
-        margin-left: 0px;
         padding-left: 0px;
```

Comment:
Also not in scope, and it has no production code impact, but my personal preference is to use `0` instead of `0px`. This is just my OCD speaking, so disregard this if you have a different opinion 😄